src/wp-includes/classicpress/class-cp-customization-frontend.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * ClassicPress Frontend Customization class
+ *
+ * @package ClassicPress
+ * @subpackage admin
+ * @since CP-2.3.0
+ */
+
+class CP_Customization_Frontend {
+	public function __construct() {
+		add_action( 'init', array( $this, 'disable_emojis' ) );
+	}
+
+	public function disable_emojis() {
+		if ( get_option( 'disable_emojis', 0 ) !== '1' ) {
```

Comment:
This should be Yoda style ideally:
`if ( '1' !== get_option( 'disable_emojis', 0 ) ) {`

Reply:
Done. Thank you for spotting.